client/src/plugins/version-info/VersionInfo.less

```diff
@@ -7,6 +7,11 @@
 
     vertical-align: super;
 
+    border: 1px solid var(--color-blue-205-100-45);
```

Comment:
C[ontinuing a conversation I've had with @smbea recently](https://github.com/camunda/camunda-modeler/pull/2658#discussion_r781027716), can we promote the new way of using color variables in our components when possible? 

That would be cool 🙂

Reply:
Ups - sure thing!

Done, thanks!